### PR TITLE
Added two basic COC & CONTRIBUTING pages

### PR DESCRIPTION
**Pull request by sarah11918**
- [ X] New or updated content

Adds 2 new pages to the repository: CONTRIBUTING.md and CODE_OF_CONDUCT.md

COC: copied from withastro/astro. We can discuss the possibility of an automated mirroring to replace this.

CONTRIBUTING: Only a basic skeleton of a page to start, BUT we needed a place to document the new `Aside` usage without the main README getting cluttered.

(Also updated README section on contributing with a note to consult CONTRIBUTING.md for specific items of note. This will get better!  😅 )

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Contributor Manual
+
+We welcome contributions of any size and skill level. As an open source project, we believe in giving back to our contributors and are happy to help with guidance on PRs, technical writing, and turning any feature idea into a reality.
+
+> **Tip for new contributors:**
+> Take a look at [https://github.com/firstcontributions/first-contributions](https://github.com/firstcontributions/first-contributions) for helpful information on contributing
```
Comment (delucis) [inline CONTRIBUTING.md:6]: We can use the more succinct link syntax here

```suggestion
> Take a look at <https://github.com/firstcontributions/first-contributions> for helpful information on contributing
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+This document is a work in progress! Most information on contributing can be found in the main README.md file. But, we need a place to begin to document some of our tools and processes, like our `Aside` Markdown feature.
+
+## Writing Asides (aka how not to abuse `blockquote`)
+
+This is an accessible component, based on the [recommended markup from the BBC’s GEL design system](https://bbc.github.io/gel/components/breakout-boxes/#recommended-markup).
+
+The component has **note**, **tip**, **caution** and **danger** variants with colour, iconography, and default labelling distinct for each.
+
+This is an Astro integration/remark plugin that allows us to use a simple syntax to use the component in Markdown and also avoid needing to import it in the frontmatter all the time.
```
Comment (delucis) [inline CONTRIBUTING.md:16]: Here’s a little more context for aside to try to guide people on when to reach for it.

```suggestion
Sometimes in documentation you want to provide information that is complementary but not strictly part of the current text or call out something that is particularly important. For example, maybe you want to include a tip that isn’t essential but could be helpful or warn a reader about a potential pitfall.

For these use cases you can use our aside component. This is an accessible component, based on the [recommended markup from the BBC’s GEL design system](https://bbc.github.io/gel/components/breakout-boxes/#recommended-markup).

The component has **note**, **tip**, **caution** and **danger** variants with colour, iconography, and default labelling distinct for each.

You can use a simple custom syntax to use the component in Markdown and also avoid needing to import it in the frontmatter all the time.
```

File: README.md [hunk 3]
```diff
@@ -58,6 +58,8 @@ Please only add new text content to the docs **in English**, only modifying **`.
 
 We have automated systems in place for notifying our community translators that there is new material to be translated, so there is no need to make changes to additional languages yourself. 
 
+Please see CONTRIBUTING.md for other specific items to note when contributing to the documentation.
```
Comment (delucis) [inline README.md:61]: We could add a link here

```suggestion
Please see [CONTRIBUTING.md](CONTRIBUTING.md) for other specific items to note when contributing to the documentation.
```

**Review by delucis — COMMENTED** [review]
Thanks for this Sarah! I’d suggest we also move this content out of our i18n guide to CONTRIBUTING.md and leave just a link in the i18n guide:

https://github.com/withastro/docs/blob/dacd3ca28f9343b0ee5593f686d01ffa1accc8dc/src/i18n/README.md?plain=1#L88-L119

I think it should be fairly straightforward to add a job to our nightly CI that checks for updates in the COC in the main repo and updates ours here if it finds any. That way we’re sure to keep them in sync.

**Comment by sarah11918** [comment]
@delucis - did a bit more reorganizing. I'm gonna have to get even basic Style Guide info up sooner rather than later, so this looks like SOMETHING.... but thoughts?

**Comment by delucis** [comment]
LGTM, Sarah! I’d say merge and improve as we go unless you have something specific you’d like to get in before that.

**Pull request merged**